modules/extension/xsd/xsd-core/src/main/java/org/geotools/xml/Parser.java

```diff
@@ -570,7 +572,8 @@ protected SAXParser parser(boolean validate) throws ParserConfigurationException
         //set hte property to map namespaces to schema locations
         parser.setProperty("http://apache.org/xml/properties/schema/external-schemaLocation",
             schemaLocation.toString());
-
+        //add the handler as a LexivalHandler too.
```

Comment:
Formatting ;-)

Reply:
also spelling LexicalHandler, and hte -> the (a couple of lines up) if you're going to make edits anyway.